src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathSingleFilter.java

```diff
@@ -0,0 +1,63 @@
+package com.puppycrawl.tools.checkstyle.filters;
+
+import com.puppycrawl.tools.checkstyle.TreeWalkerAuditEvent;
+import com.puppycrawl.tools.checkstyle.TreeWalkerFilter;
+import com.puppycrawl.tools.checkstyle.api.AutomaticBean;
+import com.puppycrawl.tools.checkstyle.api.CheckstyleException;
+
+/**
+ * 
```

Comment:
Again, since this is a wrapper of XpathFilter look to https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/filters/XpathFilter.java for how some basic documentation is done.
It will probably be similar but not exactly the same.